### ISSUE TITLE
[web][a11y] text editing does not with more than one text field

### ISSUE THREAD

**Issue by nturgut** [flutter/flutter#72894]
This is observed in all the form factors (mobile and desktop a11y) and with all browser engines (blink, webkit, gecko).

Text editing works when there is one text field. However moving between two text fields does not work. Also for the mobile web virtual keyboard stays on the page after the user finished editing.
